.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,25 @@
+name: Node CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: windows-2019
```

Comment:
Is there any reason to run this on Windows? I'd recommend using a Linux image unless there's a specific reason to use Windows

Reply:
Yes. We've got a very solid CI established for Linux with Circle CI. AppVeyor is terrible and needs to go, and experimenting with Github Actions for that purpose is a double bonus.